src/EnergyPlus/EconomicTariff.cc

```diff
@@ -482,11 +485,67 @@ namespace EconomicTariff {
 				tariff( iInObj ).energyConv = 9.4781712e-9;
 				tariff( iInObj ).demandConv = 0.00003412;
 			} else {
-				tariff( iInObj ).convChoice = conversionKWH;
-				tariff( iInObj ).energyConv = 0.0000002778;
-				tariff( iInObj ).demandConv = 0.001;
-				ShowWarningError( RoutineName + CurrentModuleObject + "=\"" + cAlphaArgs( 1 ) + "\" invalid data" );
-				ShowContinueError( cAlphaFieldNames( 3 ) + "=\"" + cAlphaArgs( 3 ) + "\", Defaulting to KWH." );
+
+
+				// TODO: is this needed?
```

Comment:
The Tariff is connected to a specific meter that has a resource so strictly it is probably not required but for clarity I think it would make sense to have specific units for the IP water units. Normally we use ft3 or gal for water units but I understand the utilities tend to use CCF. I guess I would recommend we use CCFH20.